### PR TITLE
Code smell fix

### PR DESCRIPTION
**Pull request by beke07**
Fix the code smells in our code.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 1]
```diff
@@ -38,8 +38,7 @@ public boolean fireTorpedo(FiringMode firingMode) {
 
     boolean firingSuccess = false;
 
-    switch (firingMode) {
-      case SINGLE:
+    if(firingMode == firingMode.SINGLE) {
```
Comment (daunera) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java:41]: Nice catch bro

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 2]
```diff
@@ -74,14 +73,11 @@ public boolean fireTorpedo(FiringMode firingMode) {
...
           }
         }
-        break;
-
-      case ALL:
+    }
+    else if(firingMode == firingMode.All){
         // try to fire both of the torpedo stores
 
 	firingSuccess = true;
```
Comment (daunera) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java]: WTF is this, alwasy true???

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 */
 public class TorpedoStore {
 
-  private double FAILURE_RATE = 0.0;
+  private double failureRate = 0.0;
```
Comment (daunera) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:10]: Nájsz name for the failure

**Review by daunera — COMMENTED** [review]
It was a very-very fine fixation :+1:

File: src/main/java/hu/bme/mit/spaceship/GT4500.java (hunk 2 above)
Comment (beke07) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: Oh, thanks for asking, i forgot this. I'll fix it.

**Review by daunera — APPROVED** [review]

**Pull request merged**